packages/wonder-blocks-testing/src/harness/__tests__/types.flowtest.tsx

```diff
@@ -59,6 +59,7 @@ import type {
 
 //> should error if the config type is wrong
 // 45 is not a string
+// @ts-expect-error: Type '45' is not assignable to type 'string'.
```

Comment:
**note:** Both of these cases should have been erroring - but the error suppressions got removed. This adds them back now that the types are working correctly.